cob_hardware_config/cob4-2/config/docking_linear_nav.yaml

```diff
@@ -16,7 +16,7 @@ goal_threshold_rot: 0.08 #0.08
 speed_threshold_rot: 0.01 #0.08
 
 #tf parameters
-global_frame: station_reflector_link #/map
+global_frame: odom_combined #/map
```

Comment:
I guess it only makes a difference when there is a chance to accumulate larger odometry errors that get corrected via localization.

Thus, if the goal you try to move to via linear nav is relative to `odom_combined`, this should be fine. Also, if the tf `odom_combined->station_reflector_link` (through whatever chain) is fixed, then for this case there is no difference at all.

The only real difference will probably be when running with bad odometry, over a larger distance, targeting a fixed frame in `map`.

I don't know how this is actually working, thus, I cannot say ;-)